proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -61,6 +62,25 @@ message Resource {
 enum EventType {
   EVENT_TYPE_UNSPECIFIED = 0;
   EVENT_TYPE_ORGANIZATION_CREATED = 1;
+  EVENT_TYPE_ORGANIZATION_DELETED = 2;
+  EVENT_TYPE_ORGANIZATION_MEMBER_ADDED = 3;
+  EVENT_TYPE_ORGANIZATION_MEMBER_ROLE_CHANGED = 4;
+  EVENT_TYPE_ORGANIZATION_MEMBER_REMOVED = 5;
+  EVENT_TYPE_REPOSITORY_CREATED = 6;
+  EVENT_TYPE_REPOSITORY_DELETED = 7;
+  EVENT_TYPE_REPOSITORY_PUSHED = 8;
+  EVENT_TYPE_REPOSITORY_MEMBER_ADDED = 9;
+  EVENT_TYPE_REPOSITORY_MEMBER_ROLE_CHANGED = 10;
+  EVENT_TYPE_REPOSITORY_MEMBER_REMOVED = 11;
+  EVENT_TYPE_REPOSITORY_VISIBILITY_CHANGED = 12;
+  EVENT_TYPE_PLUGIN_CREATED = 13;
+  EVENT_TYPE_PLUGIN_DELETED = 14;
+  EVENT_TYPE_PLUGIN_PUSHED = 15;
```

Comment:
Let's wait [to confirm](https://www.notion.so/Audited-Events-List-db1e266e05ce4d0db48638ec6091b9b2?d=4c291e06bee24fa1b84ef82de04ce13e#f4b78a2eac69445f994807f7d4fa1dfc) if those events make sense for plugins.

Reply:
want me to pull them out for now then?

Reply:
Nah, let's leave them and fix when needed.